set.mm

```diff
@@ -46776,6 +46793,19 @@ practical reasons (to avoid having to prove sethood of ` A ` in every use
       UDUEJUFUHUFUDUDUEQZHUDUGHUHIUDUEKUGUIUDUCABLMUDUGNOPUCABRST $.
   $}
 
+  $( Implication of a class difference with a subclass.  (Contributed by AV,
+     3-Jan-2022.) $)
+  ssdifim $p |- ( ( A C_ V /\ B = ( V \ A ) ) -> A = ( V \ B ) ) $=
+    ( wss cdif wceq dfss4 eqcom sylbb difeq2 eqcomd sylan9eq ) ACDZBCAEZFZACNEZ
+    CBEZMPAFAPFACGPAHIOQPBNCJKL $.
+
+  $( Symmetric class differences for subclasses.  (Contributed by AV,
+     3-Jan-2022.) $)
+  ssdifsym $p |- ( ( A C_ V /\ B C_ V )
```

Comment:
This looks like ssconb. See if shorter proof.

Reply:
Although these theorems look similar, I do not see any direct relationship (regarding to use the one theorem in the proof of the other) between them. But maybe I should move up ~ssdifim and ~ssdifsym near to ~ssconb.

Reply:
PS: not possible to move up ~ssdifim and ~ssdifsym , because they depend on ~dfss4.

Reply:
Ah yes, I missed the inclusions instead of equalities